packages/terra-collapsible-menu-view/src/CollapsibleMenuView.scss

```diff
@@ -15,18 +15,26 @@
     align-items: center;
     display: flex;
     flex: 0 0 auto;
-    padding-left: 0.714rem;
+    padding-left: var(--terra-collapsible-menu-view-menu-button-padding-left, 0.714rem);
 
     &.hidden {
       display: none;
       width: 0;
     }
+
+    .menu-button-icon {
+      background: var(--terra-collapsible-menu-view-menu-button-background, inline-svg('<svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="#1c1f21" d="M24 18.6a5.4 5.4 0 1 1-5.4 5.4 5.38 5.38 0 0 1 5.4-5.4zm18.6 0a5.4 5.4 0 1 1-5.4 5.4 5.38 5.38 0 0 1 5.4-5.4zm-37.2 0A5.4 5.4 0 1 1 0 24a5.38 5.38 0 0 1 5.4-5.4z"/></svg>'));
```

Comment:
In the unlikely event we add a background property to the menu button it might be safer to scope this into `menu-button-icon-background`

Reply:
`-menu-view-menu-`

![image](https://user-images.githubusercontent.com/6720991/42770163-f5b47510-88e9-11e8-85f9-ce6b2b88b18e.png)